content/v2.0/reference/config-options.md

```diff
@@ -1,193 +1,696 @@
 ---
 title: InfluxDB configuration options
 description: >
-  Configure InfluxDB when starting the `influxd` service.
-  This article outlines the different configuration options available.
+  Customize your InfluxDB configuration by passing configuration flags with the
+  [`influxd` daemon](/v2.0/reference/cli/influxd/), setting environment variables,
+  or defining configuration options in a configuration file.
 menu:
   v2_0_ref:
     name: Configuration options
     weight: 3
 products: [oss]
+related:
+  - /v2.0/reference/cli/influxd
 ---
 
+Customize your InfluxDB configuration by using configuration flags with the
+[`influxd` daemon](/v2.0/reference/cli/influxd/), setting environment variables,
+or defining configuration options in a configuration file.
+
+### Configuration precedence
+InfluxDB honors configurations settings using the following precedence:
+
+1. `influxd` daemon flags
+2. Environment variables
+3. Settings defined in a configuration file
+
+### InfluxDB configuration file
+When `influxd` starts, it checks for a **`config.yaml` in the current working directory**.
+To customize the file name or path of the configuration file, set the `INFLUXD_CONFIG_FILE`
```

Comment:
Is there no way to set this via a command line flag? `influxd --config-path /path/to/my/config.yml`?

Reply:
the 1.x version of this is INFLUXDB_CONFIG_PATH

Reply:
In talking with @jsteenb2, the presents the chicken and the egg problem.

Reply:
yah its a doozie. @russorat made a card for this. It'll likely require a larger refactor of our `cli` pkg to make this a reality. bit of a bummer. Was hoping to include this in the initial work but was just to much headache